src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/LanguageVersionExtensions.cs

```diff
@@ -10,7 +10,7 @@ public static bool IsCSharp15OrAbove(this LanguageVersion languageVersion)
         => languageVersion >= LanguageVersion.Preview;
 
     public static bool IsCSharp14OrAbove(this LanguageVersion languageVersion)
-        => languageVersion >= (LanguageVersion)1400; // Update to LanguageVersion.CSharp14 when available.
+        => languageVersion >= LanguageVersion.CSharp14;
```

Comment:
I think we can't make that change without updating a nuget package reference first.